src/Components/animation/AnimationEditor.js

```diff
@@ -342,14 +409,33 @@ class AnimationEditor extends React.Component {
     this.saveAnimation(newAnimation);
   };
 
+  removeStep = () => {
+    // Add the element with its initial position
+    var newAnimation = this._copyAnimation();
+
+    /* If the last step is currently displayed */
+    if (this.currentStep === this.state.animation.stepCount() - 1)
+      Animated.timing(this.currentStepAV, {
+        toValue: this.state.animation.stepCount() - 2,
+        duration: 0,
+        easing: Easing.linear,
+        key: 0,
+      }).start();
+
+    newAnimation.removeStep();
+
+    this.saveAnimation(newAnimation);
+  };
+
   displayStepDescription = () => {
     // TODO
   };
 
-  render() {
-    debug('render AE');
-    debug('this.currentStep: ' + this.currentStep);
+  displayStepDescription = () => {
```

Comment:
This is a duplicate of the above method?

Maybe we can remove them since we have no description for now?

Reply:
Yes it is a duplicate. Ok we can remove them.